### ISSUE TITLE
Update RangePolicy constructors

### ISSUE THREAD

**Issue by nliber** [kokkos/kokkos-core-wiki#519]
From Release 4.3:

https://github.com/kokkos/kokkos/pull/6845

While this is a breaking change, given that `ChunkSize` was the only thing that could be passed as `Args...`, it might be better not to mark this as a 4.3 specific change (it breaks a case that is unlikely to have been encountered, and adding in the previous version makes the docs more noisy for no clear benefit).

**Comment by ajpowelsnl** [comment]
@nliber - are all of the RangePolicy topics covered #523 ?  If so, can this issue be closed?  If not, can you summarize what else needs to be done to complete?

**Comment by ajpowelsnl** [comment]
@nliber - ping; what's left to on this guy?
